test/parallel/test-uv-errno.js

```diff
@@ -47,7 +47,21 @@ function runTest(fn) {
                  `Received ${err}`
       });
   });
+
+}
+
+function errNameTest(fn) {
+  // uv.errname should not cause crash with invalid args
```

Comment:
Why do you expect it to not crash? Almost everything on `process.binding()` crashes when we supply invalid args.

Reply:
I was under a more "resilient" approach, but I fully agree; it does not make sense to avoid crashing at this point.

It looks like `ErrName` is only used for calling `uvbinding.errname()`; do you all think it is time to get rid of this? Like, do the complete runtime deprecation or something.